vsphere/tests/test_check.py

```diff
@@ -73,7 +73,9 @@ def test_external_host_tags(aggregator, realtime_instance):
     for ex, sub in zip(expected_tags, submitted_tags):
         ex_host, sub_host = ex[0], sub[0]
         ex_tags, sub_tags = ex[1]['vsphere'], sub[1]['vsphere']
-        ex_tags = [to_string(t) for t in ex_tags]  # json library loads data in unicode, let's convert back to native
+        ex_tags = [
+            to_native_string(t) for t in ex_tags
+        ]  # json library loads data in unicode, let's convert back to native
```

Comment:
> let's convert back to native

I take this comment as a confirmation that `to_native_string` really is a better name. :-)